README.md

```diff
@@ -2,210 +2,133 @@
-
-### 3.2. Quality control
-
-After running the analysis, check your Quality Control (QC) report by opening the file <PATH-OUTPUT>/qc/index.html. Use the "search" feature of the QC report to quickly jump to segmentations or labeling issues.
 
-**1. Segmentations**
+### Getting the spinal cord segmentation
 
-If segmentation issues are noticed while checking the quality report, proceed to manual correction using the procedure below:
+To segment a single image, run the following command: 
 
-* In QC report, search for "deepseg" to only display results of spinal cord segmentation
-* Review segmentation and spinal cord.
-* Click on the F key to indicate if the segmentation is OK ✅, needs manual correction ❌ or if the data is not usable ⚠️ (artifact). Two .yml lists, one for manual corrections and one for the unusable data, will automatically be generated.
-* Download the lists by clicking on **Download QC Fails*** and on **Download QC Artifacts**.
-
-Proceed to manual correction using FSLeyes or ITK snap. Upload the manual segmentations (_seg-manual.nii.gz) with json sidecar in the derivatives.
-Re-run the analysis: [Launch processing](#launch-processing)
-
-**2. Registrations**
-
-* In QC report, search for "sct_register_multimodal" to only display results of registration.
-* Click on the F key to indicate if the registration is OK ✅, needs to be excluded ❌ or if the data is not usable ⚠️ (artifact). Two .yml lists, will automatically be generated.
-* Download the list by clicking on **Download QC Fails** and add the file names under `FILES_REG` to https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/blob/main/exclude.yml
-* Re-run the analysis: [Launch processing](#launch-processing)
+```bash
+sct_deepseg spinalcord -i <path-to-image>.nii.gz -o <path-to-output-file>.nii.gz 
+```
 
-**3. Soft segmentations**
+To segment a single image AND generate a QC report for the image, run the following command: 
 
-* In QC report, search for "softseg" to only display results of spinal cord soft segmentation
-* Click on the F key to indicate if the soft segmentation is OK ✅, needs manual correction ❌ or if the data is not usable ⚠️ (artifact). Two .yml lists, one for manual corrections and one for the unusable data, will automatically be generated.
-* Download the list by clicking on **Download QC Fails**.
-* Upload the soft segmentations under `data-multi-subject/derivatives/labels_softseg`.
-* Re-run the analysis: [Launch processing](#launch-processing)
+```bash
+sct_deepseg spinalcord -i <path-to-image>.nii.gz -o <path-to-output-file>.nii.gz -qc qc -qc-subject <name-of-subject>
+```
 
+More information on input arguments and their usage can be found [here](https://spinalcordtoolbox.com/stable/user_section/command-line/deepseg/seg_sc_contrast_agnostic.html).
 
-## 4. Training
+## 2. Retraining the contrast-agnostic model 
```

Comment:
just call it 'Training the model'

Reply:
Done! https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/138/commits/02917609a38185654c65d69ca6872c213adfe0ac